LibTrixi.jl/src/LibTrixi.jl

```diff
@@ -31,12 +31,24 @@ export trixi_nelements,
 export trixi_nelements_global,
        trixi_nelements_global_cfptr,
        trixi_nelements_global_jl
+export trixi_ndofs,
+       trixi_ndofs_cfptr,
+       trixi_ndofs_jl
+export trixi_ndofs_global,
+       trixi_ndofs_global_cfptr,
+       trixi_ndofs_global_jl
+export trixi_ndofs_element,
+       trixi_ndofs_element_cfptr,
+       trixi_ndofs_element_jl
```

Comment:
I understand the need to export these functions. Have you considered whether it might be more helpful to call them `trixi_ndofs`, `trixi_ndofsglobal`, and `trixi_ndofselement` instead of using `_` in the middle? Since they names are _very_ similar to the original functions used in Trixi.jl, I fear that it might lead to confusion that sometimes it's with an underscore and sometimes it isn't.

Reply:
Should it rather be `trixi_ndofselement` or `trixi_ndofscell`? (something like this does not exist in Trixi)